lib/h3core.js

```diff
@@ -581,6 +581,15 @@ export function h3GetFaces(h3Index) {
  */
 export function h3GetResolution(h3Index) {
     if (typeof h3Index !== 'string') {
+        if (
+            Array.isArray(h3Index) &&
+            h3Index.length === 2 &&
+            Number.isInteger(h3Index[0]) &&
+            Number.isInteger(h3Index[1])
+        ) {
+            const [lower, upper] = h3Index;
+            return H3.h3GetResolution(lower, upper);
+        }
```

Comment:
Shouldn't we just delegate all paths to the C implementation here? i.e.
```
const [lower, upper] = h3IndexToSplitLong(h3Index);
return H3.h3GetResolution(lower, upper);
```
I'm not sure why we implemented it in pure JS at first, but I'm not convinced it's any faster.

Reply:
Yeah, I think it's best to use a similar optimization strategy to other bindings rather than relying on puling characters out. Using the library implementation seems more predictable.